code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -2,6 +2,12 @@
 	real_name = "Test Dummy"
 	status_flags = GODMODE|CANPUSH
 
+/mob/living/carbon/human/dummy/mannequin/New()
+    ..()
+    if (!initialized)
+        args[1] = TRUE
+        SSatoms.InitAtom(src, args)
```

Comment:
At some point I should create a macro for this, there's one or two other things that could be initialized this way as well.